src/tools/tests/manifest2proto-test.ts

```diff
@@ -1459,7 +1459,7 @@ describe('manifest2proto', () => {
   // and deserialized in Kotlin to the extent that they are present in the .textproto file.
   it('encodes the Manifest2ProtoTest manifest', async () => {
     assert.deepStrictEqual(
-      await encodeManifestToProto('java/arcs/core/data/testdata/Manifest2ProtoTest.arcs'),
+      await encodeManifestToProto('java/arcs/core/data/testdata/Manifest2ProtoTest.arcs', 'java/arcs/core/data/testdata/Manifest2ProtoTest.arcs'),
```

Comment:
this test currently fail, but i'm not sure what it is trying to test - the policy in `Manifest2ProtoTest.arcs` file doesn't really match the Schema used in the recipe, so the type is restricted to null.
shall i change the Type of the policy target?

also a new question rises - how do we deal with schemas that have multiple names? currently i'm just defaulting to the first name, but is it really the right behavior?